tutorials/Land/Soil/Coupled/equilibrium_test.jl

```diff
@@ -424,101 +391,74 @@ boundaries = [
     FT(1) FT(1) zmax
```

Comment:
we can remove these lines now, since they were specific to the interpolation grid.